conf/defaults.ini

```diff
@@ -844,7 +844,7 @@ min_interval = 10s
 [unified_alerting.screenshots]
 # Enable screenshots in notifications. This option requires a remote HTTP image rendering service. Please
 # see [rendering] for further configuration options.
-enabled =
+capture =
```

Comment:
```suggestion
capture = false
```

Reply:
shouldn't it be false by default here too? 🤔 I think the unified alerting one didn't have a default in the `ini` for other reasons.

Reply:
The default value when no value is set is already `false`, so it should be fine.

Reply:
I added it anyway, to have a more explicit configuration.